indico/modules/events/registration/templates/display/participant_list.html

```diff
@@ -11,6 +11,14 @@
     {% endif %}
 {% endblock %}
 
+{% block description %}
+    {%- if tables|length > 1 -%}
+        {% trans -%}
+            The lists of participants grouped by the registration form they used to registered to the event.
```

Comment:
I added the description on the page because all you see when you load it is a title and a list of users. How does the user know where the title refers to (registration form)?